compiler/nimconf.nim

```diff
@@ -238,6 +238,23 @@ proc getSystemConfigPath*(conf: ConfigRef; filename: RelativeFile): AbsoluteFile
     if not fileExists(result): result = p / RelativeDir"etc/nim" / filename
     if not fileExists(result): result = AbsoluteDir"/etc/nim" / filename
 
+const nimRepoDirCT = currentSourcePath / ".." / ".." # this should be exposed somewhere
```

Comment:
As I explained, I don't like it and I don't understand why it's required.

Reply:
fixed; I've removed it now.
(long answer: that was to also support the case where the local development nim binary is in some location other than `nim/bin/`, however I now realize that this cannot work because rest of code assumes it's in a specific directory of depth (=1) relative to nim root dir)